resources/js/Pages/Results.vue

```diff
@@ -309,6 +312,22 @@
             showSubmitConfirmation( item: string ) {
                 this.lastSubmitted = item;
             },
+            hasChanged(entityId: string) {
+                for (var decisionId in this.decisions[entityId]) {
+                    const decision = this.decisions[entityId][decisionId];
+                    if(decision.review_status !== decision.previous_status) {
+                        return true;
+                    }
+                }
+
+                return false;
+            },
+            storeOldDecisions(item: string) {
```

Comment:
here too storePreviousDecisions for consistency. Sorry, just noticed this one :sweat_smile: